__tests__/numbers.test.js

```diff
@@ -190,19 +190,19 @@ describe('/numbers', () => {
     });
   });
 
-  describe('GET /remainder?a={number}&b={number}', () => {
-    xit('gives the remainder of dividing 18 by 5', done => {
+  describe('POST /remainder', () => {
+    it('gives the remainder of dividing 18 by 5', done => {
       request(app)
         .post('/numbers/remainder')
-        .query({ a: 18, b: 5 })
+        .send({ a: 18, b: 5 })
```

Comment:
Do you know why it needs to be send instead of query?

Reply:
Because .post is associated with .send? Get is associated with .query. Correct me if I'm wrong please!